<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Roundrobin_tab</class>
 <widget class="QWidget" name="Roundrobin_tab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="cmb_games">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Game:</string>
   </property>
  </widget>
  <widget class="QTableView" name="tab_matrix">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>431</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tab_game">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>221</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="comp_NE">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>360</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Compute NE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="comp_paropt">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>360</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Compute par opt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_repl_dyn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>360</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Replicator</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>473</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Pareto optimal:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listW_NEs">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>390</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="resizeMode">
    <enum>QListView::Fixed</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="listW_NE">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="listW_paropt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>491</y>
     <width>431</width>
     <height>71</height>
    </rect>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
